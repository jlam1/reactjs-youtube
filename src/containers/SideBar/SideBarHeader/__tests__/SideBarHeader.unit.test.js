import React from 'react';
import {shallow} from 'enzyme';
import {SideBarHeader} from '../SideBarHeader';

describe('SideBarHeader', () => {

	test('Renders SideBarHeader with props.title=null', () => {
		const wrapper = shallow(
		  <SideBarHeader />
		);
		expect(wrapper).toMatchSnapshot();
	  });
	
	  test('Renders SideBarHeader with propr.title=\'\'', () => {
		const wrapper = shallow(
		  <SideBarHeader title=''/>
		);
		expect(wrapper).toMatchSnapshot();
	  });
	
	  test('Render SideBarHeader with Sample Title', () => {
		const wrapper = shallow(
		  <SideBarHeader title='Sample Title'/>
		);
		expect(wrapper).toMatchSnapshot();
	  });

});