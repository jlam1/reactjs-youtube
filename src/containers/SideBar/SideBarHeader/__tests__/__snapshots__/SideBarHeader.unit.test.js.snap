// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarHeader Render SideBarHeader with Sample Title 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  >
    SAMPLE TITLE
  </MenuHeader>
</MenuItem>
`;

exports[`SideBarHeader Renders SideBarHeader with propr.title='' 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;

exports[`SideBarHeader Renders SideBarHeader with props.title=null 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;
